.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
Flake8 is a code quality checker that ensures we are writing good code. It will run automatically in Travis CI and not allow a pull request merge if it fails.